content/doc/book/installing/_kubernetes.adoc

```diff
@@ -181,7 +181,7 @@ $ echo http://$NODE_IP:$NODE_PORT/login
 ----
 3. Login with the password from step 1 and the username: admin
 4. Use Jenkins Configuration as Code by specifying configScripts in your values.yaml file.
-   See the configuration as code link:http:///configuration-as-code[documentation] and  https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos[examples].
+   See the configuration as code https://plugins.jenkins.io/configuration-as-code/#documentation[documentation] and  https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos[examples].
```

Comment:
There is a markup for plugins described in the [styleguide](https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#references-to-plugins) that simplifies these types of references
```suggestion
   See the plugin:configuration-as-code[configuration as code documentation] and  https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos[examples].
```